src/displays/display-show-keys.html

```diff
@@ -0,0 +1,306 @@
+                margin-top: 10px;
+            }
+
+            .keybox[wide-layout] .greylink {
+                margin-top: 0px;
+            }
+
+            .key {
+                @apply --small-bold;
+                color: var(--sc-grey3);
+                word-break: break-all;
+                text-align: center;
+            }
+
+            .keybox[wide-layout] .key {
+                text-align: left;
+            }
+
+            .subtitle {
+                @apply --smallest-light;
+                color: var(--sc-grey3);
+                margin-top: 10px;
+            }
+
+            .keybox[wide-layout] .subtitle {
+                margin-top: 0px;                
+            }
+
+
+            .bluelink {
+                @apply --blue-link;
+                margin-top: 15px;
+            }
+
+            .keybox[wide-layout] .bluelink {
+                margin-top: 10px;
+            }
+
+            .close {
+                @apply --xmark-grey2-small;
+                margin: 10px 0px 0px 0px;
+            }
+
+            .keybox[wide-layout] .close {
+                @apply --xmark-grey2-small;
+                margin: 0px 0px 0px 25px;
+            }
+
+            .close:active {
+                @apply --button-active;
+            }
+
+            .txtbox {
+                @apply --layout-vertical;
+                @apply --layout-start-justified;
+                @apply --layout-center;
+            }
+            .keybox[wide-layout] .txtbox {
+                @apply --layout-vertical;
+                @apply --layout-start-justified;
+                @apply --layout-start;
+                box-sizing: border-box;
+            }
+
+            .extramargin {
+                margin-top: 15px;
+            }
+
+            .keybox[wide-layout] .extramargin {
+                margin-top: 10px;
+            }
+            @media only screen and (-webkit-min-device-pixel-ratio: 1.5),
+            only screen and (min--moz-device-pixel-ratio: 1.5),
+            only screen and (min-resolution: 240dpi) {
+                .close,
+                .keybox[wide-layout] .close {
+                    @apply --retina;
+                }
+            }
+
+        </style>
+        <iron-media-query query="(min-width: 600px)" query-matches="{{wide}}"></iron-media-query>
+        
+        <div id="keyBox" class="keybox" wide-layout$="{{wide}}">
+            <template is="dom-if" if="{{toggleKeys}}">
+                <div class="greylink" on-click="_togglePubKey">{{localize('show my keys')}}</div>
+            </template>
+
+            <template is="dom-if" if="{{!toggleKeys}}">
+                <div class="txtbox">
+                    <div class="subtitle" >{{localize('Your public key')}}</div>                    
+                    <div class="key">{{publicKey}}</div>
+                    <template is="dom-if" if="{{!togglePK}}">          
+                        <div class="bluelink">{{localize('show private key')}}</div>
```

Comment:
This no longer open the password overlay

Reply:
You're right. I wanted to ask Xardas for help on this, but I completly forgot, I'm sry